### ISSUE TITLE
Should this library be deprecated or moved to purescript-web?

### ISSUE THREAD

**Issue by thomashoneyman** [purescript-contrib/purescript-js-timers#18]
When reviewing the timers API I discovered that the [relevant MDN documentation is marked obsolete](https://developer.mozilla.org/en-US/docs/Archive/Add-ons/Code_snippets/Timers) and that functions like `setInterval` are now part of the [WindowOrWorkerGlobalScope](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope) mixin, which covers the window object and web workers API.

With that in mind, it seems like the [web-html](https://github.com/purescript-web/purescript-web-html) library should perhaps have the additional window functions `setTimeout`, `setInterval`, etc. added to them, and this library should be deprecated.

Or am I missing a reason why these functions should remain defined here?

**Comment by garyb** [comment]
Hmm, interesting one. Moving them to `purescript-web` doesn't quite seem right, as that would suggest that they're only available in browsers, but these functions work in node also. 🤔

**Comment by thomashoneyman** [comment]
Another option might be adding these (and other missing functions) to web-html and then moving these to the Node organization under the relevant name.

...or just keeping them here would be ok as well; this just jumped out to me as being a bit awkward. What do you think?

**Comment by garyb** [comment]
Yeah, I'm not sure really. I guess the web version could re-export these along with providing the other functions perhaps. Although the functions there should perhaps take a `WindowOrWorkerGlobalScope` value, if the context they runs in also matters. I don't know about that though, cross worker/window stuff isn't something I've had to deal with very often.

**Comment by thomashoneyman** [comment]
From what I can tell, `Window` and `WorkerGlobalScope` both separately implement these functions; I don't really know anything about `WorkerGlobalScope` but it seems safe enough to at least have `Window` export these (and other) functions as there is no `WorkerGlobalScope` in -web at this time.
